transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/core/MessagingClient.kt

```diff
@@ -154,4 +154,11 @@ interface MessagingClient {
      */
     @Throws(IllegalStateException::class)
     fun disconnect()
+
+    /**
+     * Reset current [conversation] and history request pagination index.
+     * Note! After calling this function, [fetchNextPage] will request the
+     * latest available history.
+     */
+    fun clearConversation()
```

Comment:
Since one of the side-effects is that the latest available history will be requested, I'm wondering if there's a better word than "clear" as I wonder if "clear" gives the impression that the conversation history will be empty after calling. "resetConversation", "reloadConversation", "burnItAllToTheGroundAndBeginAgain"? Not sure what's most appropriate but thought I'd open the discussion.